src/tests/compatibility-tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("ClassLibrary1")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Hewlett-Packard")]
+[assembly: AssemblyProduct("ClassLibrary1")]
+[assembly: AssemblyCopyright("Copyright © Hewlett-Packard 2018")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
```

Comment:
Can this file be deleted?

Reply:
Sure... as you can see, it got added for me when I created the project. I'm pretty surprised to see that HP has the copyright. It's an HP laptop I'm using but I wonder where VS got that template from.

Reply:
That always was really silly IMO. Here's what I found: https://stackoverflow.com/a/200791

Reply:
Fascinating! The vendor sets it up so that anything you do gets copyright to them. That takes balls.

Reply:
I think the theory was that in corporate scenarios, a group policy sets that name to your organization's name. But I'd bet that ended up being the exception rather than the rule.